client/devdocs/design/search-collection.jsx

```diff
@@ -65,7 +65,9 @@ const Collection = ( {
 		}
 
 		const exampleName = getComponentName( example );
-		const exampleLink = `/devdocs/${ section }/${ camelCaseToSlug( exampleName ) }`;
+		const exampleLink = `/devdocs/${ section }/${ encodeURIComponent(
+			camelCaseToSlug( exampleName )
```

Comment:
For folks reading, this is needed for the forward slash in names like `core/heading`.